azure/packages/test/scenario-runner/README.md

```diff
@@ -4,6 +4,36 @@
 
 This package provides a reference application that executes pre-set scenarios with the intent of measuring the Fluid Framework's performance and reliability. We primarily use this package as part of a pipeline scheduled to run periodically to measure, log, and report various performance and reliability metrics. These metrics can then be used to gauge an undrestanding of the expected behavior/performance of the Fluid Framework in these various scenarios, which can help define the SLA.
 
+<!-- AUTO-GENERATED-CONTENT:START (LIBRARY_PACKAGE_README_HEADER) -->
+
+<!-- prettier-ignore-start -->
+<!-- NOTE: This section is automatically generated using @fluid-tools/markdown-magic. Do not update these generated contents directly. -->
+
+**IMPORTANT: This package is experimental.**
```

Comment:
The PR changes are correct today, but I think maybe we should end up making this package `@fluid-private`. @sashasimic do you know if we have any users other than ourselves? I'd expect not...